sentinelhub/config.py

```diff
@@ -69,132 +92,61 @@ class SHConfig:  # pylint: disable=too-many-instance-attributes
         "aws_secret_access_key",
         "aws_session_token",
     )
-    OTHER_PARAMS = (
-        "sh_base_url",
-        "sh_auth_base_url",
-        "geopedia_wms_url",
-        "geopedia_rest_url",
-        "aws_metadata_url",
-        "aws_s3_l1c_bucket",
-        "aws_s3_l2a_bucket",
-        "opensearch_url",
-        "max_wfs_records_per_query",
-        "max_opensearch_records_per_query",
-        "max_download_attempts",
-        "download_sleep_time",
-        "download_timeout_seconds",
-        "number_of_download_processes",
-    )
 
-    def __init__(self, profile: str = DEFAULT_PROFILE, *, use_defaults: bool = False):
+    def __init__(self, profile: str = DEFAULT_PROFILE, *, use_defaults: bool = False, **kwargs: Any):
         """
         :param profile: Specifies which profile to load form the configuration file. The environment variable
             SH_USER_PROFILE has precedence.
```

Comment:
I feel like the `$` makes it more intuitive

```suggestion
        :param profile: Specifies which profile to load from the configuration file. The environment variable
            $SH_USER_PROFILE has precedence.
```

Reply:
why? :thinking:

Reply:
mah.. I looked at it again now and maybe not, it's even OS specific.